docs/2.getting-started.md

```diff
@@ -1,12 +1,153 @@
 # 2. Getting Started
 
-## Installation Options
+## Installation
 
-Currently, the Abilities API is available as a WordPress plugin. You can install it by cloning the repository into your `wp-content/plugins` directory.
+Until the Abilities API is merged into WordPress core, it must be installed before it can be used.
+
+### As a plugin
+
+The best and easiest way to try and use the Abilities API is to install it as a plugin by downloading the latest release from the [GitHub Releases page](https://github.com/WordPress/abilities-api/releases/latest).
+
+#### With WP-CLI
+
+```bash
+wp plugin install https://github.com/WordPress/abilities-api/releases/latest/download/abilities-api.zip
+```
+
+#### With WP-Env
+
+```jsonc
+// .wp-env.json
+{
+  "$schema": "https://schemas.wp.org/trunk/wp-env.json",
+  // ... other config ...
+  "plugins": [
+    "WordPress/abilities-api",
+    // ... other plugins ...
+  ],
+  // ... more config ...
+}
+```
+
+#### With Composer
+
+Until the plugin is available on Packagist, you will need to add the repository to your `composer.json` file.
+
+```jsonc
+{
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/WordPress/abilities-api.git"
+    }
+    // ... other repositories.
+  ],
+  "extra": {
+    "installer-paths": {
+      // This should match your WordPress+Composer setup.
+      "wp-content/plugins/{$name}/": [
+          "type:wordpress-plugin"
+      ]
+      // .. other paths.
+    }
+  }
+  // ... rest of your composer.json.
+}
+```
+
+Then, require the package in your project:
+
+```bash
+composer require wordpress/abilities-api
+```
+
+### As a dependency
+
+Plugin authors and developers may wish to rely on the Abilities API as a dependency in their own projects, before it is merged into core. You can do that in one of the following ways.
+
+#### As a Plugin Dependency (Recommended)
```

Comment:
Even without it being on .org (and therefore available to autoinstall), I believe this should _still_  be `Recommended` , since we want to ensure people are testing with the latest versions for the next few months, and not some potentially-deconflicted library.

But if you disagree, drop it.